src/Blockcore.Indexer/Storage/Types/SyncRichListInputs.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Blockcore.Indexer.Storage.Types
+{
+   class SyncRichListInputs
+   {
+      public long value { get; set; }
+      public string address { get; set; }
+
+      public SyncRichListInputs(string address, long value)
```

Comment:
Should avoid constructors like these after new object construction was added to C#.

new SyncRichListInputs { Value = value, Address = value }; when you create the instance.

Reply:
Thanks, will note it for the future. Will remove SyncRichListInputs.cs since its not needed anymore anyways.